src/components/ProjectImagesViewer.js

```diff
@@ -1,11 +1,32 @@
 import PropTypes from 'prop-types';
+import { useEffect, useState } from 'react';
+import close from '../icons/close.svg';
+
+function ProjectImagesViewer({ styles, images, isViewerOpen, setIsViewerOpen }) {
+  const [heroImage, setHeroImage] = useState();
+
+  useEffect(() => {
+    setHeroImage(images[0])
+  }, [images]);
 
-function ProjectImagesViewer({ isViewerOpen, setIsViewerOpen }) {
   if (!isViewerOpen) return null;
 
   return (
-    <div onClick={() => setIsViewerOpen(!isViewerOpen)}>
-      ProjectImagesViewer
+    <div className={styles['image-viewer-background']}>
+      <div className={styles['image-viewer-div']}>
+        <div className={styles['image-viewer-close-div']}>
+          <button className={styles['image-viewer-close']} onClick={() => setIsViewerOpen(!isViewerOpen)}><img src={close} /></button>
+        </div>
+        <div className={styles['image-viewer-hero-div']}>
```

Comment:
avoid "div" as a class name. It's not semantic.

Reply:
Going in on the next PR for the whole project